python/client/azure/mgmt/redhatopenshift/v2019_12_31_preview/models/_models.py

```diff
@@ -196,13 +198,15 @@ class ClusterProfile(Model):
     """
 
     _attribute_map = {
+        'pull_secret': {'key': 'pullSecret', 'type': 'str'},
         'domain': {'key': 'domain', 'type': 'str'},
         'version': {'key': 'version', 'type': 'str'},
         'resource_group_id': {'key': 'resourceGroupId', 'type': 'str'},
     }
 
     def __init__(self, **kwargs):
         super(ClusterProfile, self).__init__(**kwargs)
+        self.pull_secret = kwargs.get('pull_secret', None)
```

Comment:
When calling `get`, the value for `default` defaults to `None`, so you don't need to pass it explicitly here (also applies below).

Reply:
this file is autogenerated ;-)